clingox/tests/testing/test_ast.py

```diff
@@ -154,14 +155,15 @@ def test_assertEqual(self):
             self.assertEqual(parse_literal("a"), parse_term("a"))
         expected_msg = textwrap.dedent(
             """\
-                {'ast_type': 'Literal', 'sign': 0, 'atom': {'ast_type':[93 chars] 0}}} != {'ast_type': 'SymbolicTerm', 'symbol': 'a'}
-                + {'ast_type': 'SymbolicTerm', 'symbol': 'a'}
-                - {'ast_type': 'Literal',
-                -  'atom': {'ast_type': 'SymbolicAtom',
-                -           'symbol': {'arguments': [],
-                -                      'ast_type': 'Function',
-                -                      'external': 0,
-                -                      'name': 'a'}},
-                -  'sign': 0}"""  # noqa: E501 pylint: disable=line-too-long
+                "ast.Literal(location=LOC,\\n            sign=0[269 chars]0)))" != "ast.SymbolicTerm(location=LOC,\\n             [31 chars]ue))"
+                - ast.Literal(location=LOC,
+                ?     ^ ^  ^^
+                + ast.SymbolicTerm(location=LOC,
+                ?     ^^^^^^ ^^  ^
+                +                  symbol=Function('a', [], True))-             sign=0,
```

Comment:
Do you know why there is no newline here?

Reply:
No, that is produced by the standard ```assertEqual()```

May be that has a bug?

Reply:
Try this code

```python
from unittest import TestCase
from clingox.testing.ast import parse_literal, parse_term
from clingox.pprint import pformat

class ASTTestCase(TestCase):
    """
    TestCase class for testing ASTs.
    """

    def test_a(self):
        lit = parse_literal("a"),
        term = parse_term("a")
        lit_repr = pformat(lit, hide_location=True)
        term_repr = pformat(term, hide_location=True)
        print(lit_repr)
        print(term_repr)
        self.assertEqual(lit_repr, term_repr)
```
It has the problem using only standard classes.
The formatted string looks fine

Reply:
Hmm, I suspect the output would look nicer if we append a newline to the result of `pformat()` and then compare the strings.

Reply:
Yes, it does. I'll patch it.